pkg/registration/register/aws_irsa/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
+- amrcoder
+- jaswalkiranavtar
```

Comment:
since jaswalkiranavtar is the owner of the upper dir, it is not necessary to put the name here.

Reply:
addressed. @mikeshng please approve

Reply:
Looks good. Thanks!